### PR TITLE
[ENTESB-14342]: Remove Camel-K 0.3.4 runtime from Syndesis

### PR DESCRIPTION
**Pull request by christophd**
- Remove integration-bom-camel-k
- Remove integration-runtime-camel-k
- Remove Camel-K server controller
- Remove Camel-K CRD addon
- Remove Camel-K cluster role/role bindings
- Remove Camel-K integration test runtime
- Remove syndesis CLI command for Camel-K
- Remove  Camel-K operation install
- Remove Knative connector and docs

### CODE REVIEW

File: tools/misc/syndesisCamelVersion.sh [hunk 1]
```diff
@@ -27,19 +27,13 @@ fi
...
     app/pom.xml \
     app/extension/bom/pom.xml \
-    app/integration/bom-camel-k/pom.xml \
     app/integration/bom/pom.xml \
     install/operator/build/conf/config.yaml \
-    install/operator/pkg/generator/assets_vfsdata.go \
-    ${current_camel_catalog}
-
-git mv ${current_camel_catalog} ${new_camel_catalog}
+    install/operator/pkg/generator/assets_vfsdata.go
```
Comment (christophd) [inline tools/misc/syndesisCamelVersion.sh:36]: @claudio4j could you please have a look at this particular change in this file? not sure if I did the right thing here

**Comment by christophd** [comment]
@lgarciaaco @phantomjinx could you please have a first look at the operator/installation related changes I have made to remove Camel-K runtime from Syndesis?

File: tools/misc/syndesisCamelVersion.sh (hunk 1 above)
Comment (claudio4j) [reply tools/misc/syndesisCamelVersion.sh:36]: This is ok.

**Comment by claudio4j** [comment]
There is a reference of camel-k-runtime `org.apache.camel.k:camel-knative:@camel-k-runtime.version` in
app/connector/knative/src/main/resources/META-INF/syndesis/connector/knative.json

**Comment by christophd** [comment]
@claudio4j yes thanks, I am currently in discussion with peers if we can remove the Knative component due to the fact that it is referencing Camel-K 0.3.4

**Comment by lgarciaaco** [comment]
Related fuse-online-install PR https://github.com/syndesisio/fuse-online-install/pull/250

**Review by claudio4j — APPROVED** [review]

**Pull request merged**